--- ./res/bnd/build-defaults.bnd.orig	2024-05-01 11:07:38.804582327 +0300
+++ ./res/bnd/build-defaults.bnd	2024-05-01 11:17:08.857295279 +0300
@@ -13,7 +13,7 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-Bundle-Version: ${version_cleanup;${version}}
+Bundle-Version: ${version}
 Bundle-License: https://www.apache.org/licenses/LICENSE-2.0.txt
 
 Specification-Title: Apache Tomcat
